src/templates/zh-CN/2019/table_of_contents.html

```diff
@@ -2,7 +2,8 @@
 
 {% block title %}目录 | Web Almanac 2019{% endblock %}
 
-{% block description %}2019 Web Almanac 表单内容, 列出以下部分: 页面内容, 用户体验, 内容发布, 内容分部.{% endblock %}
+{% block description %}2019 Web Almanac 表单内容, 列出以下部分: 页面内容, 用户体验, 内容发布, 内容分发。{% endblock %}
```

Comment:
This the correct comma to use?

```suggestion
{% block description %}2019 Web Almanac 表单内容, 列出以下部分: 页面内容, 用户体验、内容发布、内容分发。{% endblock %}
```